src/server/actions/study-job.actions.ts

```diff
@@ -159,20 +177,22 @@ export const onFetchStudyJobsAction = memberAction(async (studyId) => {
         .execute()
 }, z.string())
 
-export const fetchJobResultsCsvAction = memberAction(async (jobId: string): Promise<string> => {
+export const fetchJobResultsCsvAction = userAction(async (jobId): Promise<string> => {
+    await checkUserAllowedJobView(jobId)
```

Comment:
one thing im curious about re: implementation on the `xAction` wrappers, is if we followed the same pattern as this `checkUserAllowedNounVerb` for the authentication checks, ie rather than wrapping the method we would simply just call something like `await validateUserAction()` at the top of the method, very similar to these calls where we're validating. 

for me personally its a little confusing to keep track of both, i understand they're different checks - no need to address this on the PR but wanted to highlight it and maybe spark a conversation around the implementation of the server action auth checks, i think when we merged that work things were a bit rushed and didn't have time to fully process it 🤔